submissions/rubyAdeniregun/Database/Queries.sql

```diff
@@ -0,0 +1,30 @@
+
+/* Question 1: Query to find the total number of passengers that survived 
+The total number of passengers who survived = 290. */
+SELECT * FROM accident_records WHERE passenger_survived LIKE 1; 
+
+
+/* Question2: Query to find the total number of passengers who did not survive
+The total number of passengers who did not survive = 424. */
+SELECT * FROM accident_records WHERE passenger_survived LIKE 0; 
```

Comment:
The survived sheet has a total of `891` meaning if I sum up the result from `Question 1 and 2` it should total to `891`

Reply:
SO, I noticed the reason for this was because of the `INT` you specified of `age` is your `registered_passengers` table. Kingly update it to use `VARCHAR`. Then delete all records from your DB and reimport the files again and run the queries again

Reply:
> SO, I noticed the reason for this was because of the `INT` you specified of `age` is your `registered_passengers` table. Kingly update it to use `VARCHAR`. Then delete all records from your DB and reimport the files again and run the queries again

wow, that is so true.  I have been battling with this. the data actually imported 741 records for that table instead of 891 as a result of that error.  I will make the adjustments asap.

Reply:
update your previous SQL file. It needs to be part of this PR